src/library_syscall.js

```diff
@@ -93,7 +93,7 @@ var SyscallsLibrary = {
 #if ASSERTIONS
       assert(SYSCALLS.varargs != undefined);
 #endif
-      var ret = {{{ makeGetValue('SYSCALLS.varargs', 0, 'i32') }}};
+      var ret = {{{ makeGetValue('+SYSCALLS.varargs', 0, 'i32') }}};
```

Comment:
Please add a comment here that the + is for closure compiler (as otherwise it's not obvious why it is necessary).

Also, is there perhaps an annotation here as well, instead of adding code?

Reply:
Do you know why our own use of closure compiler internally doesn't seem to need this?